code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm

```diff
@@ -506,6 +493,7 @@
 		iron_content += 0.005 * delta_time
 	if(iron_content > 0 && power_level <= 4 && prob(25 / (power_level + 1)))
 		iron_content = max(iron_content - 0.01 * delta_time, 0)
+	iron_content = clamp(iron_content, 0, 1)
```

Comment:
?

Reply:
i clamped this because the iron_content was going over 100% and it caused heavy delaminations all around the board (it was meant to be hard, but it was almost impossible to handle)